test/Target/ExportSystemC/basic.mlir

```diff
@@ -10,6 +10,15 @@
 emitc.include <"systemc">
 emitc.include "nosystemheader"
 
+// CHECK-EMPTY:
+// CHECK-LABEL: SC_MODULE(submodule) {
+systemc.module @submodule (%in0: !systemc.in<i32>, %in1: !systemc.in<i32>, %out0: !systemc.out<i32>) {
+// CHECK-NEXT: sc_core::sc_in<sc_dt::sc_uint<32>> in0;
```

Comment:
This emission is fine. In the future it may be good to get this in a more human written format which would probably import everything from sc_core and sc_dt. It may be worth asking @drom or @mslijepc what they think about this.

Reply:
`using namespace sc_core & sc_dt` is certainty a good idea. If not introducing lots of dependencies, `include "systemc.h"` would solve both

Reply:
Yes, my plan is to import `systemc.h` instead of `systemc`. I wasn't aware of the difference initially.